tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-torch181-cuda102.yml

```diff
@@ -3,7 +3,7 @@ trigger: none
 stages:
 - template: templates/py-packaging-training-cuda-stage.yml
   parameters:
-    build_py_parameters: --enable_training --update --build
+    build_py_parameters: --enable_training --update --build --enable_training_torch_interop --disable_unit_tests
```

Comment:
Please add "--cmake_extra_defines onnxruntime_BUILD_UNIT_TESTS=OFF" there. Then the changes in build.py are not needed.

Reply:
Ok.